scripts/bootstrap.sh.in

```diff
@@ -44,6 +46,16 @@ EOF
 
 exec > >(tee -ia "${LOGFILE}") 2>&1
 
+determine_os() {
+    # This way permit to determine All the recent OS like Centos, Ubuntu
+    # and Debian
+     if [ -f /etc/os-release ]; then
```

Comment:
What if this file does not exist ? No fallback ? No error ?
To me, you should at least return a non-zero exit code with an explicit error message to make the script fail.

Reply:
from what I read about it, all new system integrate it, it's not present on all other oldest system, but always here on the new ones.

Reply:
I add a `die "Impossible to determine the OS"`

Reply:
If we want to fully use run_quiet(), you should rather do something like:
```
echo "Impossible to determine the OS" 1>&2
return 1
```